### ISSUE TITLE
[datafeeder] map viewer link should point to a nice configurable viewer

### ISSUE THREAD

**Issue by landryb** [georchestra/georchestra#3391]
in the last step of publication, the 'map viewer' link opens the builtin gs OL view (eg https://$fqdn/geoserver/$workspace/wms/reflect?LAYERS=$layername&width=800&format=application/openlayers#toggle) - i know you see where i'm going..

ideally, the default should point to mapstore (when it allows to arbitrary load layers from an external catalog, but in our case we can safely assume that the local gs is present in the default map config) and i know some ppl will want to use sviewer URLs or even mapfishapp urls.. so it has to be configurable, passing `$fqdn`, `$workspace` & `$layername` params to the url..

**Comment by fvanderbiest** [comment]
You're totally right, this would be a nice enhancement, and probably an easy one too !

**Comment by f-necas** [comment]
Links at the end of legacy datafeeder are configurable in his last version
